docs/userguide.rst

```diff
@@ -251,13 +253,19 @@ The above example configures 'somename.eth' to resolve to the address of your pr
 Transferring a name
 -------------------
 
-You can transfer ownership of a name you own in the ENS registry to someone else using `setOwner`:
+You can transfer ownership of a name you own in the ENS registry to another trusted account using `setOwner`:
```

Comment:
OTOH, what I was trying to achieve here is catching someone who just jumped into the docs and [searched for "transfer name"](https://veox-ens.readthedocs.io/en/doc-ens-setowner-note/search.html?q=transfer+name&check_keywords=yes&area=default), intending to actually transfer full control of it.

There are two results on that page, and the second one shows:

> For example, you can use ens.setOwner to transfer administration of the name to another

Hopefully, they'd still click the link to read the full paragraph, and then realise that `ens.setOwner()` figuring so prominently in the (un!)ellipsized text is not the contract/function they want to call.